File: skills/leak-scan/scripts/fixtures/ruby-commented/webhook_client.rb
=begin
The wire header "X-Idc-Signature" and the 300 second skew ceiling are owned by
WebhookContract. This block only explains them; restating a fact in prose must
never make prose a site.
=end
def sign(digest)
  { WebhookContract::SIGNATURE_HEADER => digest }
end

File: skills/leak-scan/scripts/fixtures/commented/webhook_client.py
"""Handles the payment webhook contract."""

from webhook_contract import SIGNATURE_HEADER


def sign(digest):
    # NOTE: the wire header is "X-Idc-Signature" and the skew ceiling is 300 -
    # both facts are owned by webhook_contract; do not restate them here.
    return {SIGNATURE_HEADER: digest}
